docs/mdbook/default.nix

```diff
@@ -196,6 +196,8 @@ with lib; let
             if isBranch
             then "${opts.index.path}/index.md"
             else "${opts.index.path}.md";
+          # Can't directly cat the moduleDoc text, so store it in a file
+          moduleDocMD = pkgs.writeText "${opts.index.path}/moduleDoc.md" opts.index.moduleDoc;
```

Comment:
If my understanding is correct, the filename passed to `writeText` doesn't need to be unique, so we can drop the `opts.index.path` prefix?

Reply:
Pushed a fixup commit that can be squashed later

Reply:
You should keep the comment you had originally added.